Dokumentation improvements (#158)
assume-framework/assume: README.md

conda activate assume-framework
```

Quick Start
-----------

> There are 3 ways to run an exemplar. <br>

Reviewer: ```suggestion
There are 3 ways to run a simulation. <br>
```

1.) local (without database and grafana) <br>
2.) with docker (with database and grafana) <br>

> 3.) use CLI to run simulations

Reviewer: I would phrase that differently.
How you run the program, and which "storage" you are using for the results are two pairs of shoes ;)
